builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -145,6 +149,14 @@ def loadServiceConfigurationData() {
       }
     }
 
+    if (service_name.trim() == "jazz_t-vault" && config_loader.TVAULT.IS_ENABLED ) {
```

Comment:
Shouldn't we do this additional check - config_loader.TVAULT.IS_ENABLED instanceof Boolean && config_loader.TVAULT.IS_ENABLED? 

FYI, I've seen this at multiple places in the PR..

Reply:
Fixed as per latest comments..